### ISSUE TITLE
segfault on enter of empty string

### ISSUE THREAD

**Issue by ksmith0** [pixie16/PixieSuite2#88]
Entering a null string and pressing enter causes a seg fault.

Here is a log of an occurrence:

```
^[[0;34mPOLL2 $ ^[[0mpwrite 0 -1 CHANNEL_CSRA 16676
  MOD 0   CHAN 0   CHANNEL_CSRA     16676
  MOD 0   CHAN 1   CHANNEL_CSRA     16676
  MOD 0   CHAN 2   CHANNEL_CSRA     16676
  MOD 0   CHAN 3   CHANNEL_CSRA     16676
  MOD 0   CHAN 4   CHANNEL_CSRA     16676
  MOD 0   CHAN 5   CHANNEL_CSRA     16676
  MOD 0   CHAN 6   CHANNEL_CSRA     16676
  MOD 0   CHAN 7   CHANNEL_CSRA     16676
  MOD 0   CHAN 8   CHANNEL_CSRA     16676
  MOD 0   CHAN 9   CHANNEL_CSRA     16676
  MOD 0   CHAN 10  CHANNEL_CSRA     16676
  MOD 0   CHAN 11  CHANNEL_CSRA     16676
  MOD 0   CHAN 12  CHANNEL_CSRA     16676
  MOD 0   CHAN 13  CHANNEL_CSRA     16676
  MOD 0   CHAN 14  CHANNEL_CSRA     16676
  MOD 0   CHAN 15  CHANNEL_CSRA     16676
Writing DSP parameters............................^[[0;32m[OK]^[[0m
^[[0;34mPOLL2 $ ^[[0mstartacq
Starting list mode run............................^[[0;32m[OK]^[[0m
Acq started on Wed Apr 20 16:08:22 2016
^[[0;34mPOLL2 $ ^[[0mstopacq
Ending run........................................^[[0;32m[OK]^[[0m
Run end status in module 0........................^[[0;32m[OK]^[[0m
Acq stopped on Wed Apr 20 16:08:34 2016
^[[0;34mPOLL2 $ ^[[0m
```

**Comment by cthornsb** [comment]
Resolved in commit b507ae4 on dev branch.

**Comment by ksmith0** [comment]
Closing as marked as resolved.
